tests/server/routes/post-metrics.js

```diff
@@ -0,0 +1,475 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!bluebird',
+  'intern/dojo/node!path',
+  'intern/dojo/node!proxyquire',
+  'intern/dojo/node!sinon',
+  'intern/dojo/node!../helpers/init-logging'
+], function (registerSuite, assert, Promise, path, proxyquire, sinon, initLogging) {
+  var mocks, route, instance;
+
+  registerSuite({
+    name: 'routes/post-metrics',
```

Comment:
nit: Seems like a lot of duplicate code but I can't think of a way to reduce it at the moment. Maybe it will come to me tomorrow :)